torch/nn/modules/_functions.py

```diff
@@ -96,3 +96,47 @@ def backward(self, grad_output):
             grad_bias = None
 
         return grad_input, grad_weight, grad_bias, None, None, None, None, None, None
+
+    @classmethod
+    def convert_sync_batchnorm(cls, module, process_group=None):
```

Comment:
hey you should add this on the module rather than the autograd function!

Reply:
Hey if I get you correctly, then you are asking to move the `convert_sync_batchnorm` function to in the `SyncBatchNorm` function here: https://github.com/pytorch/pytorch/blob/0829ef00dd5b62f7850290c722e9eaa49d45b491/torch/nn/modules/batchnorm.py#L322 Please correct me if I am wrong.